static/templates/recipient_row.hbs

```diff
@@ -39,10 +39,10 @@
 
                 {{! edit topic pencil icon }}
                 {{#if always_visible_topic_edit}}
-                    <i class="fa fa-pencil always_visible_topic_edit recipient_bar_icon" {{#unless realm_allow_message_editing}}style="display: none"{{/unless}} role="button" tabindex="0" aria-label="{{t 'Edit' }}"></i>
+                    <i class="fa fa-pencil always_visible_topic_edit recipient_bar_icon" {{#unless realm_allow_message_editing}}style="display: none"{{/unless}} role="button" tabindex="0" aria-label="{{t 'Edit' }}" title="{{t 'Edit Topic' }}"></i>
```

Comment:
Could you fix the failing `capitalization` test by changing `Edit Topic`  to  `Edit topic`. (https://zulip.readthedocs.io/en/latest/translating/translating.html#capitalization)

Reply:
Thanks Done!